core/src/main/java/org/apache/druid/java/util/http/client/response/StatusResponseHandler.java

```diff
@@ -23,16 +23,24 @@
 import org.jboss.netty.handler.codec.http.HttpResponse;
 
 import java.nio.charset.Charset;
+import java.nio.charset.StandardCharsets;
 
 /**
  */
 public class StatusResponseHandler implements HttpResponseHandler<StatusResponseHolder, StatusResponseHolder>
 {
   private final Charset charset;
```

Comment:
Please remove this field.

Reply:
@leventov , you mean to replace usage (each occurrence) of **charset** instance variable in this class directly with StandardCharsets.UTF_8

Reply:
Yes